skyvern/webeye/actions/handler.py

```diff
@@ -484,7 +484,7 @@ async def handle_select_option_action(
             return [ActionFailure(e)]
 
     try:
-        option_xpath = scraped_page.id_to_xpath_dict[action.option.index]
+        option_xpath = scraped_page.id_to_xpath_dict[action.option.id]
```

Comment:
Actually this is a legacy bug. We didn't store the option element as a single key in the `scraped_page.id_to_xpath_dict`.  So the `option.id` here will never be found.

https://github.com/Skyvern-AI/skyvern/blob/418473d72c0d867a97ac543cb7988a45f2e2f275/skyvern/webeye/scraper/scraper.py#L257-L263

Reply:
Oh yeah this is a big bug 😮 We should stop using `id_to_xpath` dict and deprecate it IMO